weave-js/src/common/components/WandbLoader.tsx

```diff
@@ -85,6 +85,8 @@ export const TrackedWandbLoader = ({
         const trackedData = {
           componentId: data.id,
           duration: data.duration,
+          start: data.start,
+          stop: data.stop,
```

Comment:
The concrete use case here is that I want to use Datadog RUM's vital metrics, which can either be given a start+stop or a start+duration. I could do this inside the wandb UI, but this seemed more correct